frontend/src/metabase/components/CopyButton/CopyButton.jsx

```diff
@@ -16,6 +16,7 @@ export default class CopyWidget extends Component {
   onCopy = () => {
     this.setState({ copied: true });
     setTimeout(() => this.setState({ copied: false }), 2000);
+    this.props.onCopy();
```

Comment:
Added a new prop of `onCopy` so we can pass in a function (generally to send a snowplow event)

Reply:
https://github.com/metabase/metabase/pull/37933